developer/language/guide/multi-platform.php

```diff
@@ -58,25 +60,26 @@ functionality is available, though there are certain additional restrictions due
   Specifies the embedded help text for web keyboards</li>
   <li><a href="/developer/language/reference/kmw_rtl" title="&amp;kmw_rtl store"><code class="language-keyman">&amp;kmw_rtl</code>:</a>
   Specifies the directionality of text controls for web keyboards</li>
-  <li><a href="/developer/language/reference/visualkeyboard"><code class="language-keyman">&amp;VisualKeyboard</code>:</a>
-  Use a store with this name to reference the .kvk file to embed into the KeymanWeb keyboard.</li>
 </ul>
 
 <h3>Recommendations</h3>
```

Comment:
```suggestion
<h3>Recommendations for Supporting Multiple Platforms</h3>
```

Since the previous section was KMW only, maybe some visual clues this section applies to supporting multiple desktops